core/server/data/meta/schema.js

```diff
@@ -69,6 +69,8 @@ function getPostSchema(metaData, data) {
     var description = metaData.excerpt ? escapeExpression(metaData.excerpt) : null,
         schema;
 
+    const context = data.page ? 'page' : 'post';
```

Comment:
Does this work with v0.1? v0.1 had no differentiation for posts & pages.
`getPostSchema` is not only used for the "data" property. It's also used if you serve a post or a page.

Reply:
It would default to `post` in that case, but will double check :+1:

Reply:
Verified and works as expected. Merging these changes